modules/ROOT/pages/setup.adoc

```diff
@@ -0,0 +1,114 @@
+= Anypoint Partner Manager Setup
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+You should be familiar with the following concepts before setting up Partner Manager:
+
+* EDI message formats and schemas.
+* Dataweave transformations.
+* Digital Certificates (such as X.509) for the AS2 transport protocol.
+* Your own network of endpoints for the listening to and sending of messages.
+* Troubleshooting in the Anypoint platform.
+
+== Minimum Requirements
+
+Verify that you have the following software and other requirements.
+
+* Mule runtime engine 4.1.1 or higher.
+
+**Note**
```

Comment:
These requirements shouldn't be hidden in notes. They are part of the Mule requirements. FYI, we do talk about vCore loads somewhere in the doc. But if the PM wants to point them straight to account rep, no problem. Just letting you know.